cheatsheets/DotNet_Security_Cheat_Sheet.md

```diff
@@ -303,7 +303,7 @@ if (string.IsNullOrEmpty(address))
  
 ### LDAP injection
 
-Almost any characters can be used in Distinguished Names. However, some must be escaped with the backslash `\` escape character. A table showing which characters that should be escaped for Active Directory can be found at the in the LDAP_Injection_Prevention_Cheat_Sheet.
+Almost any characters can be used in Distinguished Names. However, some must be escaped with the backslash `\` escape character. A table showing which characters that should be escaped for Active Directory can be found at the in the [LDAP_Injection_Prevention_Cheat_Sheet](LDAP_Injection_Prevention_Cheat_Sheet.md#introduction). 
```

Comment:
Can you remove the `_` from the url description, as they are not required.

Reply:
I removed _ from the url description.